js/components/dataset/resource/availability.vue

```diff
@@ -1,28 +1,58 @@
 <template>
-    <span v-if="availability === 'AVAILABLE'" class="badge bg-green">✓</span>
-    <span v-if="availability === 'NOT_AVAILABLE'" class="badge bg-red">×</span>
-    <span v-if="availability === 'UNKNOWN'" class="badge bg-gray">?</span>
+    <div :title="_('Click to trigger a new check')" v-if="!checking" @click.stop="refresh()">
```

Comment:
Even if available?

Reply:
It's far fetched, but maybe we want to be able to flag immediately a non available resource.

Reply:
I mean, I don't get the point of refreshing the availability of an available resource?

Reply:
This feature is meant to refresh the check status of a resource when an admin knows the status wrong. Like I said it's far fetched, but maybe we know a resource is unavailable and we want the status to reflect that immediately.

Plus, not displaying the feature when the resource is available would add some complexity and I don't think it's worth it.

I see your point though ;-)

Reply:
Ok, get it 👍